fork-common-test-dex/app/src/androidTest/java/com/shazam/forktest/ClassWithExcludedAnnotationTest.java

```diff
@@ -0,0 +1,20 @@
+package com.shazam.forktest;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import com.shazam.forktest.ExcludedAnnotation;
+
+public class ClassWithExcludedAnnotationTest {
+    @Test
+    public void includedMethod() {
+        assertEquals(4, 2 + 2);
+    }
+
+    @Test
+    @ExcludedAnnotation
+    public void excludedMethod() {
+        assertEquals(4, 2 + 2);
+    }
+
+}
```

Comment:
style: can we add an empty line at the end of the file?

Reply:
also no space before the final curly?

Reply:
should be find after d6bfea2